newdle/client/src/components/creation/FinalStep.js

```diff
@@ -45,6 +50,15 @@ export default function FinalStep() {
     }
   }
 
+  async function editNewdle() {
+    const newdle = await _editNewdle(activeNewdle.code, {title, private: isPrivate, notify});
+
+    if (newdle) {
+      history.push('/new/success');
```

Comment:
shouldn't we go back to the summary page after editing?

Reply:
In principle, we would and it worked but there is no feedback for the user. The options were:
- Create new bottom right dialogs for success like we do for errors
- Re-use the creation page.

Reply:
Showing a message on the page where you saved the edit and only redirecting when closing it would be another option